### DISCUSSION TITLE
Delete a private repository

### DISCUSSION THREAD

**Discussion by ghevoni** [community/community#66056] [New to GitHub]
Hi,

Please help me to delete a private repository, which it isn't mine and I lost connection with the owner.
I don't understand why it so difficult to remove such a repository from my account.
In my account I want to see only my repositories.
Can anyone to give me a solution for this problem?
Thank you!

**Comment by pulkitshringi** [comment]
This might be helpful : https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/removing-yourself-from-a-collaborators-repository

**Reply by ghevoni** [reply]
Thank you. It worked.
